server/routes/api/place.py

```diff
@@ -15,9 +15,10 @@
 import collections
 import json
 
-from flask import Blueprint, request
-
+from flask import Blueprint, request, jsonify
```

Comment:
Per https://google.github.io/styleguide/pyguide.html#224-decision, I think we

`import flask`

and then use `flask.Blueprint`, `flask.request`, and `flask.jsonify`.  This eliminates injecting these types into the file's namespace.  It makes it easier to understand the source of these values, which is helpful when updating code.